### ISSUE TITLE
Throwing an Error when passing empty array to `helpers.ArrayElements`

### ISSUE THREAD

**Issue by c0per** [faker-js/faker#920]
### Describe the bug

When passing in an empty Array to `helpers.ArrayElements`, I'm hoping it to return an empty Array.
Instead it throws this Error:
```
Uncaught Error: Max 0 should be larger then min 1.
    at i0.number (https://cdn.esm.sh/v78/@faker-js/faker@6.3.1/deno/faker.js:2:2634645)
    at s0.arrayElements (https://cdn.esm.sh/v78/@faker-js/faker@6.3.1/deno/faker.js:4:7275)
    at <anonymous>:2:15
```

### Reproduction

Simply `faker.helpers.arrayElements([]);`

### Additional Info

I'm using version v6.3.1 on Deno.

**Comment by Shinigami92** [comment]
> ### Describe the bug
> When passing in an empty Array to `helpers.ArrayElements`, I'm hoping it to return an empty Array. Instead it throws this Error:
> 
> ```
> Uncaught Error: Max 0 should be larger then min 1.
>     at i0.number (https://cdn.esm.sh/v78/@faker-js/faker@6.3.1/deno/faker.js:2:2634645)
>     at s0.arrayElements (https://cdn.esm.sh/v78/@faker-js/faker@6.3.1/deno/faker.js:4:7275)
>     at <anonymous>:2:15
> ```
> 
> ### Reproduction
> Simply `faker.helpers.arrayElements([]);`
> 
> ### Additional Info
> I'm using version v6.3.1 on Deno.

It could take a while until we fix this due to we are working on v7
In the meanwhile please use `yourArray.length > 0 ? faker.helpers.arrayElements(yourArray) : []`

**Comment by c0per** [comment]
I looked a bit into the code. It's a rather simple fix. I'll open a PR if it's ok to merge.

**Comment by c0per** [comment]
It's #921

**Comment by Shinigami92** [comment]
I will reopen this one, so no duplicates are created...
We can close it when we release v7
Alternatively we need to release a v6.3.2 and use cherry-pick, but as this is easyly workaroundable, we could at least wait if other bugs are found

**Comment by xDivisionByZerox** [comment]
This should be fixed in the current version (v7.2.0).
Working [stackblitz](https://stackblitz.com/edit/typescript-9r5h8w?devToolsHeight=33&file=index.ts).

For this reason I will close this.
